test/python/bfloat16/test_maxpoolbackprop_nchw.py

```diff
@@ -0,0 +1,148 @@
+# ==============================================================================
+#  Copyright 2019 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ==============================================================================
+"""nGraph TensorFlow bridge MaxPoolBackprop operation test
+
+"""
+
+# Currently, this test fails with a segmentation fault
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import pytest
+import numpy as np
+import os
+
+import tensorflow as tf
+from tensorflow.python.ops.gen_nn_ops import max_pool_grad
+
+import ngraph_bridge
+
+# Test Ngraph Op MaxPoolBackprop with data format NCHW
+# TF Op:MaxPoolGrad
+
+np.random.seed(5)
+
+#Inputs
+N = 4
+C = 3
+H = 8
+W = 8
+
+output_nchw = {
+    "VALID": np.random.rand(4, 3, 3, 3).astype('f'),
```

Comment:
can these shapes be declared as a variable and then reused

Reply:
https://github.com/tensorflow/ngraph-bridge/pull/183/files#diff-710d19f83aeab00a28522c3a83df1c8fR54